homeassistant/components/webostv/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "LG webOS Smart TV",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/webostv",
-  "requirements": ["aiowebostv==0.1.1"],
+  "requirements": ["aiowebostv==0.1.1", "sqlitedict==1.7.0"],
```

Comment:
Seems a bit wild to add a whole dependency just for this 1 call. Prefer if you can do it without.

Reply:
I can do it using `sqlalchemy` which is requirement for `recorder` and `sql` integrations, is it acceptable?
This is temporary anyhow until we deprecate the YAML configuration.

Reply:
Changed to use `sqlalchemy`